etcdserver/server.go

```diff
@@ -523,7 +523,9 @@ func NewServer(cfg ServerConfig) (srv *EtcdServer, err error) {
 			MinLeaseTTL:                int64(math.Ceil(minTTL.Seconds())),
 			CheckpointInterval:         cfg.LeaseCheckpointInterval,
 			ExpiredLeasesRetryInterval: srv.Cfg.ReqTimeout(),
-		})
+		},
+		srv.consistIndex,
```

Comment:
I know the field was an existing field, it seems naming the field consistentIndex is more intuitive.